kernel/packages/shared/types.ts

```diff
@@ -1,12 +1,10 @@
-import { parseParcelPosition } from 'atomicHelpers/parcelScenePositions'
 import { Vector3Component } from '../atomicHelpers/landHelpers'
 import { QueryType } from 'decentraland-ecs/src/decentraland/PhysicsCast'
 
 export { Avatar, Profile, ColorString, WearableId, Wearable } from './profiles/types'
 
 export type MappingsResponse = {
   parcel_id: string
```

Comment:
Maybe we can even go further and drop `parcel_id` and `root_cid` and use `sceneId` in preparation of using the content server v3.